packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -359,6 +362,10 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
         </React.Fragment>;
     }
 
+    setFocus(): void {
+        document.getElementById(this.props.outputDescriptor.id+'chartContainer')?.focus();
```

Comment:
It appears the call to focus() also makes the component scroll into view.

Therefore this scrolling does not happen when there is no series selected, when there are no results or when the analysis is still running (text displayed instead of chart).

Perhaps there could be a super() implementation to give focus to the container as a fall-back? This could be used also as a fall-back for timegraph and as a default for data-tree.

Reply:
@PatrickTasse Made use of conditional statements to call focus() on the parent abstract-output-component if the focus container does not exist